libs/kameleon/src/kameleon/app_listing.clj

```diff
@@ -44,6 +44,19 @@
               (where {:app_category_app.app_category_id
                       fav_group_id_subselect})))))
 
+(defn- add-app-id-where-clause
+  [query {:keys [app-ids]}]
+  (if (seq app-ids)
+    (where query {:id [in app-ids]})
+    query))
```

Comment:
If a set of app IDs is included in the query options, the result set is limited to apps with those IDs.